### DISCUSSION TITLE
Lambda SnapStart Option

### DISCUSSION THREAD

**Discussion by neoplastic** [serverless/serverless#11565] [Q&A]
AWS just announced a new flag to reduce the `Init` phase of a Lambda function. Is there a way we can use this with the current Serverless Framework or do we need to make a pull request to put this in?

https://aws.amazon.com/blogs/aws/new-accelerate-your-lambda-functions-with-lambda-snapstart/

**Comment by flaviostutz** [comment]
I guess we don't have a native support for SnapStart yet, but for now we could extend the generated CloudFormation template and add the newer CF property to the Lambda definition:

SnapStart:
  ApplyOn: PublishedVersions

- See "extend" at the end of https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/

- See an example of the newer CF SnapStart at https://aws.amazon.com/blogs/aws/new-accelerate-your-lambda-functions-with-lambda-snapstart/
